src/type/preload.ts

```diff
@@ -120,6 +120,10 @@ export const defaultHotkeySettings: HotkeySettingType[] = [
     action: "新規プロジェクト",
     combination: HotkeyCombination(!isMac ? "Ctrl N" : "Meta N"),
   },
+  {
+    action: "全画面表示の切り替え",
```

Comment:
どうするか迷いどころですが、他は「AをB」としてるのでここも合わせたいかも･････？
```suggestion
    action: "全画面表示を切り替え",
```

ここはメニュー欄に現れるものなので、「の」にして名詞のほうが良さそうではありますが、まあ一旦揃えておくということで！